content/en/profiler/enabling/go.md

```diff
@@ -95,6 +95,12 @@ Alternatively you can set profiler configuration using environment variables:
 | `DD_VERSION`                                     | String        | The [version][8] of your service. |
 | `DD_TAGS`                                        | String        | Tags to apply to an uploaded profile. Must be a list of `<key>:<value>` separated by commas such as: `layer:api,team:intake`.   |
 
+### Showing C function calls in CPU profiles
+
+Go's CPU profiler only shows detailed information for Go code by default. If your program calls into C code, time spent running C code will be reflected in the profile, but the call stacks will only show Go function calls.
+
+You can optionally use a library such as [ianlancetaylor/cgosymbolizer][10] to add detailed C function call information to CPU profiles. Note that this library is considered experimental. At Datadog, it has been used in production with few issues. However, the library can cause (infrequent) deadlocks in programs which use C++ exceptions, or which use libraries such as `tcmalloc` that also collect call stacks.
```

Comment:
NIT: I think we should add a code snippet that shows how to use ian's library below. 

```go
import _ "github.com/ianlancetaylor/cgosymbolizer"
```

His README doesn't include an example, and in fact might confuse people into thinking that they have to call `runtime.SetCgoTraceback` themselves.